selfdrive/test/process_replay/regen.py

```diff
@@ -247,7 +296,8 @@ def regen_segment(lr, frs=None, outdir=FAKEDATA, disable_tqdm=False):
       # ensure all procs are running
       for d, procs in fake_daemons.items():
         for p in procs:
-          if not p.is_alive():
+          if not p.is_alive() and d != 'sensord':
+            # not all sensors must have events (temperatureSensor, bmx sockets)
```

Comment:
let’s just not replay the bmx msgs at all in regen

Reply:
ok, but this still holds for the temperature sensor, some routes dont have temperature events,
dont replay temp too?

Reply:
yeah that's fine. they're not used anywhere right now